docs/docs/tutorial/refresher.mdx

```diff
@@ -17,8 +17,7 @@ for the task of deploying Grafana and Prometheus to Kubernetes:
      Prometheus.
 2. Grafana
    - Another `Deployment` is required for the Grafana server.
-   - To connect to the webinterface, we also need a `Service` of type
-     `NodePort`.
+   - To connect to the webinterface, we also need a `Service`.
```

Comment:
The idea with `NodePort` here was to be able to use a mixin later on: 

https://github.com/grafana/tanka/blob/b32282ab9220ad87c70d7f7b35c5f9daa3c4160e/docs/docs/tutorial/k-lib.mdx#L117-L118

Although I get this is a bad example as it is completely redundant

Reply:
Hmm, gotcha. What about a wording, which puts less emphasis on the NodePort? That might just do the trick:

```
To connect to the webinterface, we will be using a `Service` of type `NodePort`.
```

Reply:
This has been incorporated to the PR.